### ISSUE TITLE
Native G4 vis

### ISSUE THREAD

**Issue by oschulz** [JuliaHEP/Geant4.jl#4]
Is it possible to use Geant4 native vis (e.g. OpenGL)?

**Comment by peremato** [comment]
Probably would be possible. I started simple with the priority on the simulation engine. I'll keep the issue open for later.

**Comment by oschulz** [comment]
>  I started simple with the priority on the simulation engine

I fully support that priority! :-)

In principle HepRep export should be possible with what we have though, right?

**Comment by oschulz** [comment]
I tried your G4Visualization example @peremato - really cool! Do you see this becoming a package Geant4Visualization.jl or so?

**Comment by peremato** [comment]
Yes, this was just an experiment. We could develop a genuine Julia visualization for Geant4 as a separate package.
